tests/mvrf/test_mgmtvrf.py

```diff
@@ -156,33 +130,73 @@ def execute_dut_command(duthost, command, mvrf=True, ignore_errors=False):
     return result
 
 
-def setup_ntp(ptfhost, duthost, ntp_servers):
-    """setup ntp client and server"""
-    ptfhost.lineinfile(path="/etc/ntp.conf", line="server 127.127.1.0 prefer")
-    # restart ntp server
-    ntp_en_res = ptfhost.service(name="ntp", state="restarted")
-    pytest_assert(wait_until(120, 5, 0, check_ntp_status, ptfhost, NtpDaemon.NTP),
-                  "NTP server was not started in PTF container {}; NTP service start result {}"
-                  .format(ptfhost.hostname, ntp_en_res))
-    # setup ntp on dut to sync with ntp server
-    for ntp_server in ntp_servers:
-        duthost.command("config ntp del %s" % ntp_server)
-    duthost.command("config ntp add %s" % ptfhost.mgmt_ip)
+@pytest.fixture
+def setup_ntp_server(duthosts, rand_one_dut_hostname, ptfhost, ntp_servers):
+    duthost = duthosts[rand_one_dut_hostname]
+
+    # Check if ntp was not in sync with ntp server before enabling mvrf, if yes then setup ntp server on ptf
+    # It should normally be set to lab ntp server
+    ntp_in_sync = duthost.command("ntpstat", module_ignore_errors=True)["rc"] == 0
```

Comment:
Please use the `check_ntp_status` function in `tests.common.helpers.ntp_helper` instead.